src/app/components/Toolbar/Features/Account/DerivationFormatter.tsx

```diff
@@ -12,7 +12,7 @@ export const DerivationFormatter = (props: DerivationFormatterProps) => {
   const { t } = useTranslation()
   const walletTypes: { [type in WalletType]: string } = {
     [WalletType.UsbLedger]: t('toolbar.wallets.type.usbLedger', 'USB Ledger'),
-    [WalletType.BleLedger]: t('toolbar.wallets.type.bluetoothLedger', 'BLE Ledger'),
+    [WalletType.BleLedger]: t('toolbar.wallets.type.bluetoothLedger', 'Bluetooth Ledger'),
```

Comment:
This was exclusively used in the import modal(I think). Due to layout breaking.

Reply:
"BLE" was to squeeze in import modal, but it's also displayed in settings > manage account
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/ec3c7ca5-8eb4-4a4e-b6fd-670a05053e6a)